data/quickstarts/oneapi-aikit-notebook.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: oneapi-aikit-notebook
```

Comment:
datat/applications/aikit.yaml 
~~~
 quickStart: create-aikit-notebook
~~~
This must to be equal